src/bin/util/setup.rs

```diff
@@ -29,6 +29,12 @@ use tikv::util::logger;
 
 // A workaround for checking if log is initialized.
 pub static LOG_INITIALIZED: AtomicBool = ATOMIC_BOOL_INIT;
+// Default is 128.
+// Extended since blocking is set, and we don't want to block very often.
+const SLOG_CHANNEL_SIZE: usize = 2048;
```

Comment:
I suggest using a larger one, like 10240. If TiKV has many regions and when it restarts, it will log many messages.